packages/integration-tests/src/test-project-setup/test_project_base.ts

```diff
@@ -144,4 +149,42 @@ export abstract class TestProjectBase {
 
     assert.ok(clientConfigStats.isFile());
   }
+
+  /**
+   * Verify deployed client outputs
+   */
+  async assertDeployedClientOutputs(backendId: BackendIdentifier) {
+    const { BackendOutputClientFactory: npmBackendOutputClientFactory } =
+      await import(
+        pathToFileURL(
+          path.join(
+            this.projectDirPath,
+            'node_modules',
+            '@aws-amplify',
+            'deployed-backend-client',
+            'lib',
+            'backend_output_client_factory.js'
+          )
+        ).toString()
+      );
+
+    const amplifyClient = new AmplifyClient(e2eToolingClientConfig);
```

Comment:
this should probably be moved to the constructor to keep all the client creation in the same place